src/Routes/RoutesCommon.tsx

```diff
@@ -257,19 +199,77 @@ export const routes: Route[] = [
     shortLabel: 'navBar.wallet',
     icon: <WalletIcon />,
     main: Dashboard,
-    category: RouteCategory.Wallet,
+    category: RouteCategory.Featured,
     mobileNav: true,
-    priority: 1,
+    priority: 5,
     routes: [
       {
         path: '/transaction-history',
         label: 'navBar.transactionHistory',
         icon: <TxHistoryIcon />,
         main: TransactionHistory,
```

Comment:
question - what are these for and are they even used anymore?

Reply:
Think those are just for routing purposes, they're not exposed but still present for routing.
Weird about this one in particular though, seems to be a dupe of `/activity` i.e 
<img width="1727" alt="image" src="https://github.com/user-attachments/assets/8015ed69-6d1b-496e-a5ca-23c2a7278b2d" />
<img width="1726" alt="image" src="https://github.com/user-attachments/assets/ae958b82-a819-4a3b-b394-3829a1aa367f" />

Happy to follow-up with a removal of this if it doesn't break things, since that's the theme of today haha